src/chigame/games/views.py

```diff
@@ -157,12 +158,29 @@ class TournamentCreateView(CreateView):
     # Note: we may remove the "matches" field later for the same reason,
     # but we keep it for now because it is convenient for testing.
 
+    # This method is called when valid form data has been POSTed. It
+    # overrides the default behavior of the CreateView class.
+    def form_valid(self, form):
+        response = super().form_valid(form)
+        create_tournaments_brackets(self.object)  # This should be changed later
```

Comment:
I believe there should be a way to block someone from creating a tournament if the number of players in the tournament is greater than the maximum number of players in that tournament. I was able to create a tournament with `max_players` equal to 1 and add 2 users to create a match between them.

Reply:
Hi Rafael, this is a good catch! It will be checked by the join_tournament functionality in a later pull request! Currently, the create_tournament_brackets feature only creates the brackets literally, and later we will implement a way for the users to utilize this feature.